app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = FoodsFacade.search_by_food(params[:q])
+    @hits = FoodsService.foods_by_ingredient(params[:q])[:totalHits]
```

Comment:
There is a way this could be refactored to only have to make 1 API call. It is also more conventional to have the facade retrieve data from the service (like you did on line 3)

Reply:
*1 API call for both the 10 foods and the total hits